large-message-core/src/main/java/com/bakdata/kafka/AmazonS3Client.java

```diff
@@ -49,12 +59,20 @@
 class AmazonS3Client implements BlobStorageClient {
 
     static final String SCHEME = "s3";
-    private final @NonNull AmazonS3 s3;
+    private final @NonNull S3Client s3;
 
-    private static ObjectMetadata createMetadata(final byte[] bytes) {
-        final ObjectMetadata metadata = new ObjectMetadata();
-        metadata.setContentLength(bytes.length);
-        return metadata;
+    static ObjectIdentifier asIdentifier(final S3Object s3Object) {
+        return ObjectIdentifier
+                .builder()
+                .key(s3Object.key())
+                .build();
```

Comment:
Just for consistency :) 
```suggestion
        return ObjectIdentifier.builder()
                .key(s3Object.key())
                .build();
```